packages/block-library/src/image/edit.native.js

```diff
@@ -0,0 +1,10 @@
+/** @format */
```

Comment:
Also, let's remove the Prettier pragma since Prettier is not used in the Gutenberg repo/setup.